frontend/src/components/ui-kit/components/Discount/Discount.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import styles from './Discount.module.scss';
+
+const Discount = (props) => {
+  return (
+    <div>
+      {props.isSmallCard ? (
```

Comment:
Destruct the props object, and assign isSmallCard a default value so that it doesn't have to be passed every time